From ecdee70c487316f396510c7cd3c4efe65003fac1 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 9 Oct 2015 11:30:03 +0200
Subject: [PATCH] base/dracut-lib.sh:getargs() don't return 1 for empty values

getargs() is only used with real values and not booleans.

So, return the values only.

(cherry picked from commit d1f7f2ac046c77913d1ca97cf8414d9226ae1286)
---
 modules.d/99base/dracut-lib.sh | 2 --
 1 file changed, 2 deletions(-)

diff --git a/modules.d/99base/dracut-lib.sh b/modules.d/99base/dracut-lib.sh
index 1eaf4b3..03a283c 100755
--- a/modules.d/99base/dracut-lib.sh
+++ b/modules.d/99base/dracut-lib.sh
@@ -330,8 +330,6 @@ getargs() {
     if [ -n "$_gfound" ]; then
         if [ $# -gt 0 ]; then
             printf '%s' "$*"
-        else
-            printf 1
         fi
         debug_on
         return 0
